ccloader/js/normalize.js

```diff
@@ -5,10 +5,45 @@ if (!window.require) {
 	window.isBrowser = true;
 	window.require = name => {
 		if (name === 'path') {
-			return {
+			const path = {
 				sep: '/',
-				join: (...args) => args.join('/')
+				normalize: function (path) {
+
+					if (path.length === 0) {
+						return '.';
+					}  else if (!path.includes(this.sep)) {
+						return path;
+					}
+
+					// split by the path separator 
+					let pieces = path.split(this.sep);
+
+					// resolve all .. and .
+
+					for (let pieceIndex = 0; pieceIndex < pieces.length; pieceIndex++) {
+						const currPiece = pieces[pieceIndex];
+						
+						if (currPiece === '.' || (currPiece === '..' && pieceIndex === 0) 
```

Comment:
Move `|| (currPiece === '..' && pieceIndex === 0)` and comment on each case:
1.) `//If piece is a "."`
2.) `//or the first piece is a ".."`
3.) `//or the piece is empty`